files/fare_transfer_rules_ft.md

```diff
@@ -13,8 +13,8 @@ File MUST contain the following attributes:
 
 Required Attributes	| Description										
 ----------			| -------------		
-`from_fare_class`	| An ID that identifies the `fare_class` that the passenger is coming from.  
-`to_fare_class`		| An ID that identifies the `fare_class` that the passenger is going to.  
+`from_fare_period`	| An ID that identifies the `fare_period` that the passenger is coming from.  
```

Comment:
yeah, i think that is the idea- to model time of day specific fares, including transfers. Fare_id does not give time of day specific information.